haskell/private/actions/link.bzl

```diff
@@ -319,7 +323,7 @@ def link_library_dynamic(hs, cc, posix, dep_info, extra_srcs, objects_dir, my_pk
 
     args = hs.actions.args()
     args.add_all(["-optl" + f for f in cc.linker_flags])
-    args.add_all(["-shared", "-dynamic"])
+    args.add_all(["-shared", "-dynamic", "-fPIC"])
```

Comment:
Is `-fPIC` needed on linking? IIRC GHC has some handling of these flags built-in, see [this note](https://github.com/tweag/rules_haskell/blob/5acb7432a55ff5c203043b9acdc7ab228cacebcf/haskell/private/actions/link.bzl#L195-L236).

Reply:
I could have *sworn* it didn't compile without that, but now it does. Probably something I changed after that removed the need for it.